README.adoc

```diff
@@ -3,137 +3,359 @@
+. Do we want to release based on different git repository?
+
+At the moment we identify four release type
+
+. Weekly
+. Stable
+. Security
+. Lts security
+
+[NOTE]
+====
+link:https://release.ci.jenkins.io[release.ci.jenkins.io] is designed in a way that we have two generic jobs, one for release and a second one for packaging.
+Then we have one job per release type which triggered the two generic Jenkins jobs with different parameters.
+While it isn't required to trigger a release type job, it just increase visibility and reduce human error factor.
+====
+
+=== Weekly
+
+The weekly release is the default release. It happened automatically every Tuesday as defined by this link:https://github.com/jenkins-infra/release/blob/19685def608c641496e6e2de3d40c275ca5e913d/Jenkinsfile.d/core/weekly#L15[cron].
+It uses parameters defined in this link:https://github.com/jenkins-infra/release/blob/master/profile.d/weekly[file]
+
+It releases using the repository https://github.com/jenkinsci/jenkins[jenkinsci/jenkins] from the branch master. Artifacts are pushed to the default maven repository 'Releases'.
+
+If for some reason the release job needs to be re-triggered, you can:
+
+. Connect to the Jenkins VPN
+. Open your favorite browser to link:https://release.ci.jenkins.io[release.ci.jenkins.io]
+. Review the weekly environment https://github.com/jenkins-infra/release/blob/master/profile.d/weekly[file]:
+. Trigger the weekly link:https://release.ci.jenkins.io/blue/organizations/jenkins/core%2Fweekly%2Frelease/branches/[job]
+
+[NOTE]
+====
+You can re-trigger individually the two downstream jobs, release and packaging.
+
+- Re-triggering the release will do a version bump then push new artifacts.
+- Re-triggering the packaging job won't published artifacts if they already exist but it will update website html.
+====
+
+=== Stable
+A stable release is a manually triggered release that happened around once a month.
+It uses parameters defined in this link:https://github.com/jenkins-infra/release/blob/master/profile.d/stable[file].
+
+Before triggering a new stable release, some steps are required:
+
+. Prepare `jenkinsci/jenkins` repository -> missing documentation link.
+. Create a branch on jenkins-infra/release with a branch name that match the release branch from jenkinsci/jenkins like `stable-2.135`
+. Review and update the stable environment https://github.com/jenkins-infra/release/blob/master/profile.d/stable[file] with:
+.. `RELEASE_GIT_BRANCH` set to the `jenkinsci/jenkins` release branch like `stable-2.135`
+.. `JENKINS_VERSION` set to the final release version that will be packaged. If set to 'stable' then the packaging job will try to guess the version based on what was pushed to the maven repository. cfr settings.
+.. `PACKAGING_GIT_BRANCH` set to the appropriated `jenkinsci/packaging` branch
+. Trigger the stable link:https://release.ci.jenkins.io/blue/organizations/jenkins/core%2Fstable%2Frelease/branches/[job]
```

Comment:
Use 2.235 rather than 2.135 as the example
```suggestion
. Prepare `jenkinsci/jenkins` repository -> missing documentation link.
. Create a branch on jenkins-infra/release with a branch name that match the release branch from jenkinsci/jenkins like `stable-2.235`
. Review and update the stable environment https://github.com/jenkins-infra/release/blob/master/profile.d/stable[file] with:
.. `RELEASE_GIT_BRANCH` set to the `jenkinsci/jenkins` release branch like `stable-2.235`
.. `JENKINS_VERSION` set to the final release version that will be packaged. If set to 'stable' then the packaging job will try to guess the version based on what was pushed to the maven repository. cfr settings.
.. `PACKAGING_GIT_BRANCH` set to the appropriated `jenkinsci/packaging` branch
. Trigger the stable link:https://release.ci.jenkins.io/blue/organizations/jenkins/core%2Fstable%2Frelease/branches/[job]
```